examples/gatsby-sourcing-wordpress/src/pages/index.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react"
+import { Link, graphql } from "gatsby"
+
+class Homepage extends Component {
+  render() {
+    const data = this.props.data
+
+    return (
+        <div>
+          <h1>Pages</h1>
+          {data.allWordpressPage.edges.map(({ node }) => (
+            <div key={node.slug}>
+              <Link to={node.slug}>
+                <h2>{node.title}</h2>
+              </Link>
+              <h3>{node.excerpt}<h3/>
```

Comment:
somehow many closing tags are wrong in this example.

wrong: `<h3/>` (only for 'self closing tags')
right: `</h3>`